week-5-practise/js/main.js

```diff
@@ -0,0 +1,66 @@
+//// PART-1 ////
+
+var placeToJumbotron = document.querySelector(".jumbotron");
+var placeToDonateABike = document.querySelector(".btn-primary"+".btn-lrg");
+var placeToVolunteer = document.querySelector(".btn-secondary");
+
+var blueButton = document.querySelector("#blueBtn");
+blueButton.addEventListener("click", blueButtonFunction);
+function blueButtonFunction() {
```

Comment:
Can you think of a more meaningful name for your function? Usually your function names should describe what the function does. In this case it's changing colours, right?

Reply:
Thanks. You are right. I have gotten Naming issues.